docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,669 @@
+
+Note that as of this time, `netstandardX.X` is not supported.  You'll have to use the `dotnet` monikers.  Here's a mapping:
+
+| dotnet | netstandard |
+|--------| ----------- |
+| dotnet51 | netstandard1.0 |
+| dotnet52 | netstandard1.1 |
+| dotnet53 | netstandard1.2 |
+| dotnet54 | netstandard1.3 |
+| dotnet55 | netstandard1.4 |
+
+
+The following is what you need to know about .NET Platform Standard targeting:
+
+You need to pick a version of `dotnetXX` (where `XX` is a version number) to add to your `project.json` file.
+
+1. You acn target the latest version - `dotnet55` - of the .NET Platform Standard, which will give you access to the most APIs.  `dotnet55` is supported by .NET Framework 4.6 (and higher) and .NET Core 1. 
+2. You can target any earlier version of the .NET Platform Standard to target earlier .NET platforms and versions.  Earlier .NET Platform Standard versions expose fewer APIs. Example targets:
+
+    - .NET Framework 4.6, 4.5.2, 4.5.1, or 4.5
+    - Windows Phone
+    - Windows Phone Silverlight
+    - Universal Windows Platform
+    - DNX Core
+    - Xamarin Platforms
+    - Mono 
+
+    [Refer to the platform mapping table](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#mapping-the-net-platform-standard-to-platforms) to choose the `dotnetXX` moniker you need to target.
+    
+    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `dotnet54`.
+
+    ```javascript
+    {
+        "frameworks":{
+            "dotnet54":{}
+        }
+    }
+    ```
+    
+    If you wanted to be compatible with Windows Phone Silverlight 8, you would pick `dotnet51`.  Note that this gives you the ability to target more platforms that just Windows Phone Silverlight 8, but at the cost of less available APIs.
+    
+    ```javascript
+    {
+        "frameworks":{
+            "dotnet51":{}
+        }
+    }
+    ```
+3. If you care about compatibility with the .NET Framework versions 4.0 or below, or need to use .NET Framework APIs not available in .NET Core, you'll have to read the next sections.
+
+## How to target .NET Framework
+
+**NOTE:** This requires .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+
+Keep in mind that some of the .NET Framework versions used here are no longer in support.  [Refer to this FAQ](https://support.microsoft.com/en-us/gp/framework_faq/en-us) about unsupported versions.
+
+Say you wish to support older frameworks, such as .NET Framework 4.0, so you can reach a maximum number of developers and projects.  From a project targeting perspective, this is quite simple!  You will need to specifically target the version of .NET Framework with its relevant Target Framework Monikers.  These are mapped out here:
```

Comment:
"Say" is a colloquialism. We know what it means, but it's not great for non-English speakers. How about:

> You may want to reach the maximum developers and projects, so use .NET Framework 4 as your baseline target. From a targeting ...